exe_7.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+
+
+def make_matrix(size:int=4)->np:
+    """
+
+    :param size: Size of matrix.
+    :return: Array with random values, and swapping first and last rows.
+    """
+    matrix = np.random.rand(size, size)
+    new_matrix = np.copy(matrix)
+    new_matrix[0] = matrix[-1]
+    new_matrix[-1] = matrix[0]
+    return new_matrix
+
+
+"""
+
+print(make_matrix())
+
+"""

```

Comment:
This should not be here.
Don’t use git add -a ever, use git add -p
https://git-scm.com/docs/git-add